aries-site/src/examples/components/fileinput/FileInputValidationExample.js

```diff
@@ -0,0 +1,54 @@
+import { Box, Button, FileInput, Form, FormField } from 'grommet';
+import { useState } from 'react';
+
+const exampleFiles = [
+  { name: 'file-size-okay.csv', size: 15000 },
+  { name: 'file-size-too-large.zip', size: 2550000 },
+];
+
+export const FileInputValidationExample = () => {
+  const maxSize = 2500000; // 2.5 MB
```

Comment:
Since we use 1024 as the conversion factor, we need to increase this. The validation message currently reads 2.4MB

Reply:
🤔  hmmm.... I'm curious how this is going to play out over the long term. Windows/Mac/Linux systems present these differently. Windows in binary, Mac (and Linux I believe) in decimal.

Further complicating things, storage presents these values in decimal form, whereas CPU/memory presents in binary form.

Storage, Mac, etc. have opted for decimal because it is more human friendly and speaks to a wider audience (non-computer science folks).

Lastly, some folks have been using MB (decimal) vs. MiB (binary) to differentiate the two formats (we are using MB), but this isn't a standard which has been adopted either...

This is a larger, entrenched battle we aren't going to solve. I think we need to be clear and upfront in our documentation to maxSize to help.

```
Specifies a file size limit in bytes (binary) for FileInput.
```

Reply:
Done.

Reply:
Hmm.. good analysis. I wasn't aware of that prior and would be a good addition to the discussion started by this issue: https://github.com/grommet/grommet/issues/5756